parsers/bbcontactsparser.py

```diff
@@ -0,0 +1,21 @@
+from utils.exceptions import InvalidFormat
+from operator import itemgetter
+
+
+def BbcontactsParser(input):
+    contents = input.split('\n')
+    output = []
+
+    for line in contents:
+        line = line.lstrip().split()
+        if not line or len(line) != 8:
+            continue
+        elif line[6].isdigit() and line[7].isdigit() and len(line) == 8:
```

Comment:
You are checking twice that `len(line) == 8`, first with `if not line or len(line) != 8:` and then with `elif line[6].isdigit() and line[7].isdigit() and len(line) == 8:`. Remove the later and just leave the first check.